cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -637,6 +639,14 @@ configureExOptions _showOrParseArgs src =
       (reqArg "VERSION" (parsecToReadE ("Cannot parse cabal lib version: "++)
                                     (fmap toFlag parsec))
                         (map prettyShow. flagToList))
+  , option "" ["append"]
+      "appending the new config to the old config file"
```

Comment:
What about:
```suggestion
      "appending the given configuration to the one in cabal.project.local"
```

Reply:
I avoided calling it `cabal.project.local` explicitly because it could be called something else, through the use of the `--project-file` flag, but that's probably overzealousness on my part.

Reply:
no I agree, that's a good point. Maybe "appending the given configuration to the one in the configuration file". Or feel free to leave it as is